STMicroelectronics/STM32L4_L4+/app/startup/gnu/STM32L475xx_FLASH.ld

```diff
@@ -47,9 +47,17 @@ MEMORY
 {
 RAM (xrw)      : ORIGIN = 0x20000000, LENGTH = 96K
 RAM2 (xrw)      : ORIGIN = 0x10000000, LENGTH = 32K
-FLASH (rx)      : ORIGIN = 0x8000000, LENGTH = 1024K
+FLASH (rx)      : ORIGIN = 0x8000000, LENGTH = 1018K
```

Comment:
Same for the 4S5.

Reply:
The registry sections are at the very end of bank2 of flash memory to avoid being overwritten in case of OS updates. Do we want to keep that or move it to right after the OS?

Reply:
We are not moving it right after the OS, we are moving it to the end of the space reserved for the OS, but distant enough to the end of the OS.